### ISSUE TITLE
Immutable.js version

### ISSUE THREAD

**Issue by mspoulsen** [staltz/cycle-onionify#6]
Wouldn't it be relatively simple to make an immutable.js version? I am using immutable.js for anything state-related so a cycle-onionify-immutable would be fantastic! :smiley:

**Comment by staltz** [comment]
Yes I'd guess it would be, you could attempt to do it already, just follow the existing source code

**Comment by jvanbruegge** [comment]
Made a POC in #19
